packages/repository/github/src/container.ts

```diff
@@ -19,7 +20,11 @@ export function initContainer(baseContainer: Container | undefined = undefined):
     }
 
     // Services
+    container.bind(GitRepositoryProvider).toSelf().inSingletonScope();
```

Comment:
Do not inject, it should be already injected from the given container (at least we expect for it)

Package do not manage injection from other packages